__tests__/integration/repeatableResources.test.js

```diff
@@ -36,7 +36,7 @@ describe('Adding new embedded Resource Templates', () => {
   it('clicks on a nested property with an embedded Note resource template and then clicks on the AddButton for a second resource template', async () => {
     await pupExpect(page).toClick('a[data-id=\'note\']')
     await pupExpect(page).toClick('.col-sm-4 > div:nth-child(1) > button:nth-child(1)')
-    await pupExpect(page).toMatchElement('div.panel:nth-child(1) > div:nth-child(2) > div:nth-child(1) > div:nth-child(2) > div:nth-child(3) > div:nth-child(2) > h5:nth-child(2)',
+    await pupExpect(page).toMatchElement('div.panel-body > div > div:nth-child(2) > div:nth-child(3) > div.rOutline-property > div:nth-child(2) > div.row > section.col-sm-8> h5',
```

Comment:
this selector is improved but still pretty yucky - can it be further improved to avoid "nth-child" and "col-sm-8" as a class name?

Reply:
Agree and open to suggestions. I have 0 expertise in this area.

Reply:
You probably already keep this page handy while writing tests:  https://www.w3schools.com/cssref/css_selectors.asp.

Reply:
I do now.